mail/flowmailer/build.gradle

```diff
@@ -18,14 +18,19 @@ dependencies {
     implementation project(":mail")
 
     implementation "org.springframework.boot:spring-boot-autoconfigure"
+    implementation "com.vladmihalcea:hibernate-types-52:2.12.1"
+    implementation "org.springframework.boot:spring-boot-starter-webflux"
+
 
     annotationProcessor "org.springframework.boot:spring-boot-autoconfigure-processor"
 
+    testImplementation project(":mail")
     testImplementation "org.springframework.security:spring-security-test"
     testImplementation "org.camunda.bpm.extension.mockito:camunda-bpm-mockito:4.13.0"
 
     //Kotlin
     testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0"
+    testImplementation 'org.mockito:mockito-inline:4.0.0'
```

Comment:
Denk dat dit naar de mail module moet want daar is dit een impl van.

Reply:
als ik dit verplaats zoals aangegeven dan gaat het fout:
`Cannot mock/spy class com.ritense.mail.flowmailer.config.FlowmailerProperties
Mockito cannot mock/spy because :
 - final class
org.mockito.exceptions.base.MockitoException: 
Cannot mock/spy class com.ritense.mail.flowmailer.config.FlowmailerProperties
Mockito cannot mock/spy because :
 - final class`

Reply:
Deze melding ken ik. In kotlin is alles closed hierdoor kan mockito een spy/mock uitvoeren. In de resource folder van mail zit een config file die dit gedrag omzeilen. Kopieer dit zou ik zeggen.

Reply:
config lijkt niet te werken. Misschien even in algemene doorloop samen naar kijken